src/audio.cpp

```diff
@@ -49,25 +49,15 @@ bool NxAudioEngine::fileExists(char* filename)
 bool NxAudioEngine::init()
 {
 	if (_engine.init() == 0)
-	{		
-		if (nullptr != winamp_in_plugin)
-		{
-			_winampInPlugin = new WinampInPlugin(BufferOutPlugin::instance());
-
-			if (!_winampInPlugin->open(winamp_in_plugin))
-			{
-				error("couldn't load %s, please verify 'winamp_in_plugin' or comment it\n", winamp_in_plugin);
-				delete _winampInPlugin;
-				_winampInPlugin = nullptr;
+	{
+		if (winamp_in_plugin != nullptr) {
+			if (!_openPsfPlugin.open(winamp_in_plugin)) {
+				error("NxAudioEngine::%s couldn't load %s, please verify 'winamp_in_plugin' or comment it\n", __func__, winamp_in_plugin);
```

Comment:
Is this a leftover?

Reply:
I need to rename winamp_in_plugin configuration entry. In this PR I load a DLL (openpsf.dll) dynamically at runtime, like I did before with the Winamp layer. But it will be removed with static linking.